content/docs/concepts/components/lifecycle/analyze.md

```diff
@@ -1,11 +1,16 @@
 +++
 title="Analyze"
-weight=2
+weight=1
 summary="Restores files that buildpacks may use to optimize the build and export phases."
 +++
 
 {{< param "summary" >}}
 
+The `analyzer` is responsible for analyzing the cache and the metadata from previously run images (if available) to determine what layers can or cannot be reused.
+This information is used during the `export` phase in order to avoid re-uploading unchanged layers.\
+Starting from `Platform API 0.7`, the `analyze` phase runs before the `detect` phase in order to validate registry access for all images that are used prior to the `detect` and `build` phases, and therefore provides faster failures for end users.\
```

Comment:
```suggestion
Starting from `Platform API 0.7`, the `analyze` phase runs before the `detect` phase in order to validate registry access for all images that are used during the build, and therefore provides faster failures for end users.
```

Reply:
Ohh is that `\` for adding an extra newline? TIL...

Reply:
Don't you want to emphasize that in case of a problem, starting from `platform API 0.7`, the user will get an error during the `analyze` phase instead of the `build` phase that comes after the `detect` and `analyze` phases?

Reply:
I think that's what it's saying now, no?

Reply:
Please see https://github.com/buildpacks/docs/pull/412#discussion_r706408676 for an idea on how to rewrite this.

Reply:
I like the addition of "The other responsibilities of the `analyzer` are moved to the `restorer`."

Reply:
Please see https://github.com/buildpacks/docs/pull/412/commits/74d6a4cee786e2ee6473f37d70c8711759baeacb and let me know what you think.